CHANGELOG.md

```diff
@@ -20,14 +20,37 @@ It contains new features, additional configuration options, and bug fixes.
 
 ### Upgrading from 0.7
 
+To upgrade your cluster:
+
+1. backup any persistent volumes and previous configuration, to be safe
```

Comment:
I find it quite extreme to recommend backing up of PDs, do we really want to suggest this? It is a tricky thing to do.

Reply:
I feel like it's always safe to say "we recommend backing up XXX" as long as you don't imply that you **must** back up PDs. We don't want to create unnecessary work for people, but we also don't wanna be on the hook if something goes wrong and a person gets mad at us because we never mentioned it :-)

Reply:
We should clarify that this is only helm-managed PVs (just hub state), not user PVs which are not managed by the chart, and thus won't be deleted by an upgrade.